packages/frontend/editor-ui/src/composables/useCanvasMapping.ts

```diff
@@ -327,6 +327,17 @@ export function useCanvasMapping({
 		}, {}),
 	);
 
+	const nodeExecutionWaitingForNextById = computed(() =>
+		nodes.value.reduce<Record<string, boolean>>((acc, node) => {
+			acc[node.id] =
+				node.name === workflowsStore.lastAddedExecutingNode &&
+				workflowsStore.executingNode.length === 0 &&
+				workflowsStore.isWorkflowRunning;
+
+			return acc;
+		}, {}),
+	);
+
```

Comment:
Following old pattern here, but this file will need to be refactored for performance reasons.